docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -218,29 +190,17 @@ config example using the secret you created above.
     < . . . >
 ----
 +
-* If you need to present extra files to any of the services, you can use
-`extraMounts`. For example, when using ceph, you'd need the {rhos_component_storage_file} ceph
-user's keyring file as well as the `ceph.conf` configuration file
-available. These are mounted via `extraMounts` as done with the example
-below.
-* Ensure that the names of the backends (`share_backend_name`) remain as they
-did on {OpenStackShort} {rhos_prev_ver}.
-* It is recommended to set the replica count of the `manilaAPI` service and
-the `manilaScheduler` service to 3. You should ensure to set the replica
-count of the `manilaShares` service/s to 1.
-* Ensure that the appropriate storage management network is specified in the
-`manilaShares` section. The example below connects the `manilaShares`
-instance with the CephFS backend driver to the `storage` network.
-* Prior to adopting the `manilaShares` service for CephFS through NFS, ensure that
-you have a clustered Ceph NFS service created. You will need to provide the
-name of the service as ``cephfs_nfs_cluster_id``.
+* If you need to add extra files to any of the services, you can use `extraMounts`. For example, when using {Ceph}, you can add the {rhos_component_storage_file} Ceph user's keyring file as well as the `ceph.conf` configuration file.
```

Comment:
Maybe the list starting with “If you need to add extra files…” should go under the code in the “patch the `OpenStackControl Plane” CR step? These list items seem to be about customizing the patch if needed?

Reply:
Yes, that's right.